Source/WebService/Blackjack.WebService/ClientApp/src/app/types/gamer.ts

```diff
@@ -1,88 +1,89 @@
 export abstract class Gamer {
-    /**
-     * Bool flag indicates if the player score exceeds 21
-     */
-    public hasBusted: boolean = false;
-    /**
-     * The current cards the player has
-     */
-    protected _hand: string[] = [];
-    /**
-     * The current cards the player has
-     */
-    public get hand(): string[] {
-        return this._hand;
-    }
-    /**
-     * The sum of the value of the player's hand
-     */
-    protected _score: number = 0;
-    /**
-     * The sum of the value of the player's hand
-     */
-    public get score(): number {
-        return this._score;
-    }
+  /**
+   * Bool flag indicates if the player score exceeds 21
+   */
+  public hasBusted: boolean = false;
+  /**
+   * The current cards the player has
+   */
+  protected _hand: string[] = [];
+  /**
+   * The current cards the player has
+   */
+  public get hand(): string[] {
+    return this._hand;
+  }
 
-    /**
-     * Adds cards provided to player's {@link Gamer._hand}
-     * @param cards
-     */
-    public updateHand(cards: string[]): void {
-        this._hand.push(...cards);
-        this.updateScore();
-    }
+  public status: string;
```

Comment:
Use the same protected set/public get pattern as hand and score.

Currently anything upstream can directly modify this value so there is no data integrity here.

Only the Player and Dealer class should be able to set this value and have full control over it.